paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -74,9 +80,44 @@ class CustomerSheet @Inject internal constructor(
         CustomerSessionViewModel.clear()
     }
 
+    /**
+     * Returns a flow that retrieves the customer's saved payment option selection or null if the
+     * customer does not have a persisted payment option selection or an error occurred.
+     */
+    fun retrievePaymentOptionSelection(): Flow<PaymentOptionSelection?> = flow {
+        val adapter = CustomerSessionViewModel.component.customerAdapter
+        val selectedPaymentOption = adapter.retrieveSelectedPaymentOption()
```

Comment:
I'm not sure what to expect here, but right now this will call the network multiple times (I think) if they consume this flow from multiple places, or multiple times.

This actually feels like something we want to live at the viewModel level, cache there, and only call the network once.

I think this is something we can likely follow up on (doesn't need to all happen in this PR), but I feel a bit uneasy about the current implementation.